pkg/ruler/ruler.go

```diff
@@ -150,58 +140,53 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 }
 
 // Ruler evaluates rules.
+//+---------------------------------------------------------------+
```

Comment:
Please indent pre-formatted text, so that go doc and other tools display it correctly.